src/sentry/static/sentry/app/stores/environmentStore.jsx

```diff
@@ -35,10 +35,11 @@ const EnvironmentStore = Reflux.createStore({
   },
 
   getByName(name) {
-    name = '' + name;
-    for (let i = 0; i < this.items.length; i++) {
-      if (this.items[i].name === name) {
-        return this.items[i];
+    if (typeof name === 'string') {
```

Comment:
nit/fyi, our preferred style is to avoid deep nesting with early returns when possible e.g.

```
if (typeof name !== 'string') return null;
for (....)
```